tests/cases/docker/chrome-devtools-frontend-next/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:current
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git /depot_tools
+ENV PATH=/depot_tools:$PATH
+WORKDIR /
+RUN mkdir devtools
+WORKDIR devtools
+RUN fetch devtools-frontend

```

Comment:
The instructions here and below are taken from https://github.com/ChromeDevTools/devtools-frontend/blob/master/docs/workflows.md#standalone-checkout